sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosFullTextPath.java

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Constants;
+import com.azure.cosmos.implementation.JsonSerializable;
+import com.azure.cosmos.implementation.apachecommons.lang.StringUtils;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Path settings within {@link CosmosFullTextPolicy}
+ */
+public class CosmosFullTextPath {
+    @JsonProperty(Constants.Properties.PATH)
+    private String path;
+    @JsonProperty(Constants.Properties.LANGUAGE)
+    private String language;
+    private JsonSerializable jsonSerializable;
```

Comment:
This seems to be unused (Note, I'm not well-versed in how serialization works in Java).

Reply:
Removed it, as it was not being used anywhere.